Error when building website

When running `yarn start:site`, I get

```text
/home/srmagura/Documents/oss/preconstruct/site/src/typography.js
  0:0  error  Parsing error: require() of ES Module /home/srmagura/Documents/oss/preconstruct/node_modules/eslint/node_modules/eslint-scope/lib/definition.js from /home/srmagura/Documents/oss/preconstruct/node_modules/babel-eslint/lib/require-from-eslint.js not supported.
Instead change the require of definition.js in /home/srmagura/Documents/oss/preconstruct/node_modules/babel-eslint/lib/require-from-eslint.js to a dynamic import() which is available in all CommonJS modules
```

Seems like this will solve it: https://stackoverflow.com/a/69557309/752601. PR incoming

--- a/package.json
+++ b/package.json
@@ -51,8 +51,6 @@
     "@types/source-map-support": "^0.5.0",
     "@types/terser": "^3.12.0",
     "@typescript-eslint/parser": "^5.4.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^10.0.3",
     "babel-jest": "^24.9.0",
     "babel-plugin-macros": "^2.8.0",
     "codecov": "^3.6.1",

--- a/site/src/gatsby-theme-sidebar/root.js
+++ b/site/src/gatsby-theme-sidebar/root.js
@@ -45,14 +45,16 @@ const MenuButton = (props) => (
   />
 );
 
-const Title = (props) => (
+const Title = ({ children, ...props }) => (
   <h1
     {...props}
     css={{
       margin: 0,
       fontSize: 16,
     }}
-  />
+  >
+    {children}
+  </h1>
 );
 
 const hamburger = (
@@ -68,11 +70,19 @@ const hamburger = (
 );
 
 let components = {
-  a: ({ href, ...props }) => {
+  a: ({ href, children, ...props }) => {
     if (href.startsWith("http")) {
-      return <a href={href} {...props} />;
+      return (
+        <a href={href} {...props}>
+          {children}
+        </a>
+      );
     }
-    return <Link to={href} {...props} />;
+    return (
+      <Link to={href} {...props}>
+        {children}
+      </Link>
+    );
   },
   blockquote: (props) => {
     return (
@@ -164,6 +174,7 @@ export default ({
               fontSize: 24,
               marginRight: 8,
             }}
+            aria-hidden
           >
             🎁
           </div>

